packages/@uppy/dashboard/src/components/AddFiles.js

```diff
@@ -6,7 +6,7 @@ class AddFiles extends Component {
     super(props)
 
     this.triggerFileInputClick = this.triggerFileInputClick.bind(this)
-    this.handleFileInputChange = this.handleFileInputChange.bind(this)
+    this.onHandleFileInputChange = this.onHandleFileInputChange.bind(this)
```

Comment:
`handle` and `on` kinda means the same thing in this situation, so i think this should either be `handleFileInputChange` or `onFileInputChange`